Sources/KlaviyoCore/Utils/DeepLinkHandler.swift

```diff
@@ -72,13 +74,15 @@ public class DeepLinkHandler {
 
     @MainActor
     private static func openWithFallbackHandler(url: URL) async {
-        if #available(iOS 14.0, *) {
-            Logger.navigation.warning("""
-            Attempting to handle universal link via a fallback mechanism.
-            For improved stability and future-proofing, please provide your own
-            deep link handler logic by calling `KlaviyoSDK().registerDeepLinkHandler(_:)`
-            on application launch. Refer to the Klaviyo Swift SDK's README for more details.
-            """)
+        if KlaviyoEnvironment.isReactNative {
+            if #available(iOS 14.0, *) {
```

Comment:
Just curious, is it that #available checks can't be in the same `if` statement as a runtime check?